src/components/common/navbar.tsx

```diff
@@ -1,10 +1,12 @@
-import { FC, useState } from "react";
+import { FC, KeyboardEvent, useState } from "react";
 import SephoraLogo from "../../assets/logo.svg";
 import MenuIcon from "../../assets/menu.svg";
 import SearchBar from "./searchbar";
 import CustomImage from "./customimage";
 import { useNavigate } from "react-router-dom";
```

Comment:
All library imports should be at top. Move this to line 2.

Reply:
done